### DISCUSSION TITLE
Thank you Biome!

### DISCUSSION THREAD

**Discussion by martpie** [biomejs/biome#3996] [General]
I just wanted to show some appreciation to the Biome project after migrating a project of mine (Museeks) from Stylelint, ESLint, Prettier and all the jazz to Biome, a couple of months ago.

Biome is fast, comes with extremely sensible defaults and is really easy to setup. It may not be as complete as others, but it's good enough that I am willing to ignore some missing rules and trade their absence for peace of mind.

Thank you, and kudos to the team working on Biome :)

**Comment by ematipico** [comment]
Thank you for the kind words! ☺️
